core/src/main/java/org/apache/accumulo/core/classloader/ClassLoaderUtil.java

```diff
@@ -75,13 +75,25 @@ static synchronized void resetContextFactoryForTests() {
 
   @SuppressWarnings("deprecation")
   public static ClassLoader getClassLoader(String context) {
-    if (context != null && !context.isEmpty()) {
-      return FACTORY.getClassLoader(context);
-    } else {
+    try {
```

Comment:
In talking with @keith-turner he was concerned about the change in behavior with the try-catch block. Prior to this change if `FACTORY.getClassLoader(context)` threw an error it would be thrown. With this change, it's hidden and the Accumulo classloader is returned.